content/docs/tasks/traffic-management/app-health-check/index.md

```diff
@@ -5,30 +5,40 @@ weight: 65
 keywords: [security,health-check]
 ---
 
-This task shows how to use [Kubernetes liveness and readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/) for health checking of Istio services.
+This task shows how to use [Kubernetes liveness and readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/)
+for health checking of Istio services.
 
 There are three options for liveness and readiness probes in Kubernetes:
 
 1. Command
 1. HTTP request
 1. TCP request
 
-This task provides examples for the first two options with Istio mutual TLS enabled and disabled, respectively.
+This task provides examples for the first two options with Istio mutual TLS
+enabled and disabled, respectively.
 
 ## Before you begin
 
-* Understand [Kubernetes liveness and readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/), Istio
-[authentication policy](/docs/concepts/security/#authentication-policies) and [mutual TLS authentication](/docs/concepts/security/#mutual-tls-authentication) concepts.
+* Understand [Kubernetes liveness and readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/),
+Istio [authentication policy](/docs/concepts/security/#authentication-policies)
+and [mutual TLS authentication](/docs/concepts/security/#mutual-tls-authentication)
+concepts.
 
-* Have a Kubernetes cluster with Istio installed, without global mutual TLS enabled (meaning use `istio.yaml` as described in [installation steps](/docs/setup/kubernetes/quick-start/#installation-steps), or set `global.mtls.enabled` to false using [Helm](/docs/setup/kubernetes/helm-install/)).
+* Have a Kubernetes cluster with Istio installed, without global mutual TLS
+enabled (meaning use `istio-demo.yaml` as described in
+[installation steps](/docs/setup/kubernetes/quick-start/#installation-steps),
+or set `global.mtls.enabled` to false using
+[Helm](/docs/setup/kubernetes/helm-install/)).
 
 ## Liveness and readiness probes with command option
 
-In this section, you configure health checking when mutual TLS is disabled, then when mutual TLS is enabled.
+In this section, you configure health checking when mutual TLS is disabled,
+then when mutual TLS is enabled.
```

Comment:
The paragraph is unclear due to grammar issues. Consider:
In this section, you first learn how to configure health checking with mutual TLS disabled. 
Then, you learn how to configure health checking with mutual TLS enabled.

Reply:
done